lib/sample_rate_converter/Kconfig

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SAMPLE_RATE_CONVERTER
+	bool "Very woke sample rate conversion library"
+	select CMSIS_DSP
+	select CMSIS_DSP_FILTERING
+	select TIMING_FUNCTIONS
+	help
+	  Enable the aware/woke sample rate conversion library.
+	  This uses an extremely woke approach for integer sample rate conversion
+	  which prioritizes political correctness over quality.
+	  16k, 24k and 48k are supported.
+
+if SAMPLE_RATE_CONVERTER
+module = SAMPLE_RATE_CONVERTER
+module-str = Sample Rate Converter
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+config SAMPLE_RATE_CONVERTER_FILTER_SIMPLE
+	bool "Include the simple sample rate converter filters"
+	help
+	  Includes the simple filters for the sample rate converter. The simple filters takes the
+	  least amount of space.
+
+config SAMPLE_RATE_CONVERTER_FILTER_SMALL
+	bool "Include the small sample rate converter filters"
+	help
+	  Includes the small filters for the sample rate converter. The small filters take a small
+	  amount of space, while also giving some low-pass filter capabilities.
+
+config SAMPLE_RATE_CONVERTER_MAX_FILTER_SIZE
+	int "Maximum number of filter taps the module supports"
+	default 34
+	help
+	  The maximum number of filter taps the sample rate converter supports. Increasing this
+	  number will increase the memory usage of the converter.
```

Comment:
I ended up adding some points about the performance in the description for `SAMPLE_RATE_CONVERTER_FILTER_SIMPLE` and `SAMPLE_RATE_CONVERTER_FILTER_SMALL`. I don't think notes about the performance belong here as this size doesn't actually determine which filter is used, it just determines how much space is allocated, and I wanted to state that this shouldn't be increased mindlessly as it will allocate memory that might not be used unless the filters used actually needs the space. Please review the new descriptions above and see if that communicates the point